eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -93,6 +93,15 @@ jobs:
       vmImage: '$(OSVmImage)'
 
     steps:
+    - pwsh: |
+        $setDailyDevBuild = "false"
+        if (('$(Build.Reason)' -eq 'Schedule') -and ('$(System.TeamProject)' -eq 'internal')) {
+          $setDailyDevBuild = "true"
+        }
+        echo "##vso[task.setvariable variable=SetDevVersion]$setDailyDevBuild"
```

Comment:
Nit: if I'm using powershell, I stick with powershell functions. Powershell maps `echo` to `Write-Output` as an alias, so I guess it doesn't really matter :P

Reply:
Discussed with Praveen, this is deliberately identical to the same steps in the JS repo.